README.md

```diff
@@ -16,11 +16,22 @@ A simple Terraform provider for MAAS.  This is a work in progress and by no mean
 The provider requires some variables to be configured in order to gain access to the MAAS server:
 
 - **api_version**:  This is optional and probably only works with 2.0. The defaults to 2.0.
-- **api_key**: [MAAS API Key](https://maas.ubuntu.com/docs/maascli.html#logging-in)
+- **api_key**: [MAAS API Key](https://maas.ubuntu.com/docs/maascli.html#logging-in), specify with the `MAAS_API_KEY` environment variable
```

Comment:
Nit, mostly around how I saw this being worded in other providers like https://github.com/hashicorp/terraform-provider-consul/blob/f4a8a3cc90de53e58c3f65fd0d6025fb1a0f807e/website/docs/index.html.markdown#argument-reference

You don't have to specify it with the env var, you can provide it in the module configuration too as `api_key` right? Maybe good to reword as something like:
```
Can also be specified with `MAAS_API_KEY` as an environment variable.
```

Reply:
But also if this is trying to steer folks in the right direction by default, could also be good to keep it as is 🤔

Reply:
Although some folks might opt to have terraform read secrets via another provider, like https://learn.hashicorp.com/tutorials/terraform/secrets-vault?in=terraform/security

Reply:
Lemme know if this is what you were thinking!